R/draw_elements.R

```diff
@@ -22,36 +22,46 @@ draw_elements = function(
     facet_by,
     split_data,
     i = 1,
-    xlvls,
-    lgnd_labs,
-    x_by = FALSE
+    x_by = FALSE,
+    flip = FALSE
     ) {
 
       ## polygons before lines, segments/arrows before points, etc.
       if (isTRUE(empty_plot)) {
 
       } else if (type == "ribbon") {
-        polygon(
-          x = c(xx, rev(xx)),
-          y = c(yymin, rev(yymax)),
-          col = bg[i],
-          border = FALSE
-        )
+        if (isFALSE(flip)) {
```

Comment:
I recommend adding an assertion at the very top of the function `assert_flag(flip)`. This function is already in the package (but not used yet). It will ensure `TRUE`/`FALSE` and return an informative error. Otherwise, you probably want to change this condition because graphs will be flipped whenever someone passes a string or other non-conforming value.

Reply:
Ah, nice. I'll add it to main `tinyplot.R` function since we already call `isTRUE(flip)`/`isFALSE(flip)` upstream there.